README.md

```diff
@@ -1 +1,164 @@
+
+#### Parameters
+
+* `lock_name`: *Optional.* This field is optional but goes with the `need_approval` field. It specifies which lock to fetch from the dynamodb.
+  
+* `need_approval`: *Optional.* If set, the `get` will wait indefinitely for a change on the `approved` field.
+
+
+### `out`: Claim or release a lock.
+
+Performs one of the following actions to change the state of the pool.
+
+#### Parameters
+
+* `action`: *Required.* This field has two possible values : `claim` and `release`.
+  The `claim` will create/acquire a lock and the `release` will release the lock
+
+* `lock_name`: *Required.* This field contains the name of the lock to acquire/release
+
+* `need_approval`: *Optional.* If set, the `put` will add a field to tell the `get` that the .
+
+## Example Concourse Configuration
+
+The following example pipeline models: acquiring, passing through, and releasing
+a lock based on the example git repository structure:
+
+```
+resource_types:
+- name: approval
+  type: docker-image
+  source:
+    # use tag `latest` for stable release
+    repository: cycloid/approval-resource
+    tag: test
+
+resources:
+- name: approval
+  type: approval
+  source:
+    #debug: true
+    AWS_ACCESS_KEY_ID: {{ec2_access_key}}
+    AWS_SECRET_ACCESS_KEY: {{ec2_secret_key}}
+    AWS_DEFAULT_REGION: eu-west-1
+    pool: cycloid-approval
+    
+jobs:
+  - name: test1
+    plan:
+      - get: resource-src
+        trigger: true
+        passed: [push-test-image]
+      - put: approval
+        params:
+          lock_name: test1
+          action: claim
+          need_approval: true
+
+  - name: test2
+    plan:
+      - get: approval
+        trigger: true
+        passed: [test1]
+        params:
+          lock_name: test1
+          need_approval: true
+      - get: resource-src
+        trigger: true
+        passed: [test1]
+      - put: approval
+        params:
+          lock_name: test1
+          action: release
+
+```
+
+## CLI
+
+To manage the lock, there's a little CLI script to interact with dynamodb.
+You can of course write your own tool.
+
+### List
+```
+# python3 cli.py list
+approved    claimed    description    id                                    lockname    need_approval    pipeline           pool              team    timestamp
+----------  ---------  -------------  ------------------------------------  ----------  ---------------  -----------------  ----------------  ------  --------------------------------
+            True                      f47f5864-a1b4-4bc0-8899-784ae3b768a4  test1       True             approval-resource  cycloid-approval  main    2016-11-30 13:55:05.079579+00:00
+```
+
+### Approve
+```
+# python3 cli.py approve --id f47f5864-a1b4-4bc0-8899-784ae3b768a4
+INFO: The lock f47f5864-a1b4-4bc0-8899-784ae3b768a4 has been approved
+```
+
+### Reject
+```
+# python3 cli.py reject --id f47f5864-a1b4-4bc0-8899-784ae3b768a4
+INFO: The lock f47f5864-a1b4-4bc0-8899-784ae3b768a4 has been rejected
+```
```

Comment:
Already done in https://github.com/cycloidio/approval-resource/blob/master/LICENSE